index.html

```diff
@@ -68,6 +71,8 @@ <h4 class="advantages-title">Мало туристов</h4>
           <li class="advantages-item">
             <h4 class="advantages-title">Там есть мост дьявола</h4>
             <p class="advantages-description">Да, по нему можно пройти! Если вы осмелитесь, разумеется</p>
+            <img class="advantages-photo" alt="Изображение моста Дьявола" src="images/devil's_bridge.jpg" width="
```

Comment:
В имени допустимо использовать только латинские буквы, цифры и символы `-` и `_`.
Необычные символы могут сыграть злую шутку: например, система на которой работает сервер, может их иначе обрабатывать, чем твой компьютер. И тогда картинка не покажется.

В таких случаях в имени файла можно несколько пренебречь грамматикой:
```suggestion
            <img class="advantages-photo" alt="Изображение моста Дьявола" src="images/devils_bridge.jpg" width="
```